create_blog_post_using_azure_openai_and_publish_in_medium/main.bal

```diff
@@ -0,0 +1,49 @@
+import ballerina/http;
+import ballerina/io;
+import ballerinax/azure_openai.text;
+import ballerinax/medium;
+
+configurable string openAIToken = ?;
+configurable string serviceUrl = ?;
+configurable string deploymentId = ?;
+configurable string mediumToken = ?;
+
+const API_VERSION = "2023-03-15-preview";
+
+public function main(string title) returns error? {
+    final medium:Client medium = check new ({auth: {token: mediumToken}});
+    final text:Client azureOpenAI = check new (
+        config = {httpVersion: http:HTTP_1_1, auth: {apiKey: openAIToken}},
+        serviceUrl = serviceUrl
+    );
+
+    medium:UserResponse response = check medium->getUserDetail();
+    medium:User? user = response.data;
+
+    if user !is medium:User {
+        return error("User is not a medium user");
+    }
+    string? userId = user.id;
+    if userId !is string {
+        return error("Medium user ID is not valid");
+    }
+
+    text:Deploymentid_completions_body completionsBody = {
+        prompt: string `Write a blog article on ${title} in markdown format.`,
+        max_tokens: 1000
+    };
+    text:Inline_response_200 completion = check azureOpenAI->/deployments/[deploymentId]/completions.post(API_VERSION, completionsBody);
+    string? content = completion.choices[0].text;
+
+    if content !is string {
+        return error("Generated content is not a string");
+    }
+
+    medium:Post post = {
+        title,
+        contentFormat: "markdown",
+        content
+    };
+    medium:PostResponse postResponse = check medium->createUserPost(userId, post);
+    io:println(postResponse);
```

Comment:
Do other samples have  io:println(postResponse) similar to this; what does this post response contain

Reply:
It contains the following information about the created post:
```
{"data":{"id":"9f76da196018","title":"WSO2","authorId":"15fe45afa1cd7d1358ca5f31b48b129f528c30a534aee009326727c7d36bfb7c9","tags":[],"url":"https://medium.com/@janetest152/wso2-9f76da196018","canonicalUrl":"","publishStatus":"public","publishedAt":1680173373503,"license":"all-rights-reserved","licenseUrl":"https://policy.medium.com/medium-terms-of-service-9db0094a1e0f"}}
```

Reply:
Addressed in [473866e](https://github.com/ballerina-guides/ai-samples/pull/62/commits/473866e23bff07df188f9dc25717539769f0624d) to just print the post url